<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Momentum Minecraft</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/images/icon.png</normaloff>:/image/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgba(0,0,0,0)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image:   url(:/image/images/main6.jpeg);
background-position: center;
border: 0px solid;

</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>50</y>
      <width>421</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(149, 140, 197);
border: 3px solid rgb(148,101,251);
border-radius: 17px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="titleAbout">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>30</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px solid;
text-align: center;
background: rgba(0,0,0,0);
border-bottom: 2px solid white;
border-radius: 0px;
color: white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TITLE&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="textAbout">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>401</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px solid;
background: rgba(0,0,0,0);
color: white;
 text-align: center;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>472</x>
      <y>40</y>
      <width>371</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(0,0,0,0);
border: 0px solid;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(73, 170, 62);
color:white;
font-size: 30px;
border-radius: 10px;


</string>
     </property>
     <property name="text">
      <string>ИГРАТЬ</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="versionsBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="downloadProgress">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="clientNameLine">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>360</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(149, 140, 197);
border: 2px solid rgb(148,101,251);
border-radius: 7px;
color: white;
font-size:15px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>541</y>
      <width>871</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(56, 49, 77);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>298</x>
       <y>12</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;MOMENTUM MINECRAFT ВСЕ ПРАВА ЗАЩИЩЕНЫ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-6</y>
      <width>871</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(56, 49, 77);
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>829</x>
       <y>11</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
